pages/landing/enterprise-eks/contact/_form.html

```diff
@@ -39,42 +39,35 @@ <h1 style="text-align: center;">
       <div class="form-group">
         <div class="row">
           <div class="col-xs-12 col-md-4">
-            <label>Which features are you interested in?</label>
-          </div>
-        </div>
-      </div>
-      <div class="form-group">
-        <div class="row">
-          <div class="col-xs-12 col-md-4">
-            <label>Service Mesh</label>
+            <label style="padding-top: 0;">Service Mesh</label>
           </div>
           <div class="col-xs-12 col-md-8">
-            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="istio" value="Istio" %}
-            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="consul-connect" value="Consul Connect" %}
-            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="aws-app-mesh" value="AWS App Mesh" %}
+            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="istio" value="Istio" width="145px" %}
+            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="consul-connect" value="Consul Connect" width="145px" %}
+            {% include_relative _input-checkbox.html name="interested-feature-service-mesh" id="aws-app-mesh" value="AWS App Mesh" width="160px" %}
```

Comment:
NIT: swap the order of Consul Connect and AWS App Mesh; the spacing will look a tiny bit better.

Reply:
Done: 7911607